docs/excel/excel-add-ins-ranges-set-get-values.md

```diff
@@ -0,0 +1,308 @@
+---
+title: Set and get range values, text, or formulas using the Excel JavaScript API
+description: 'Code samples that show how to use the Excel JavaScript API to set and get range values, text, or formulas.'
+ms.date: 04/02/2021
+localization_priority: Normal
+---
+
+# Set and get range values, text, or formulas using the Excel JavaScript API
+
+This article shows how to set and get range values, text, or formulas with the Excel JavaScript API. For the complete list of properties and methods that the `Range` object supports, see the [Excel.Range class](/javascript/api/excel/excel.range).
+
+[!include[Excel cells and ranges note](../includes/note-excel-cells-and-ranges.md)]
+
+## Set values or formulas
+
+The following examples show how to set values and formulas for a single cell or a range of cells.
+
+### Set value for a single cell
```

Comment:
Should we still include the single-cell variant in the new cell-and-range world? This could be the sample that goes in the "Cells" article.

Reply:
*Waiting to address this until we get feedback from Ricky.*